next-app/src/lib/icons/ImagesIcon.svelte

```diff
@@ -0,0 +1,3 @@
+<svg viewBox="0 0 24 24" class="inline-block w-7 h-7 fill-current">
```

Comment:
I'm commenting here because I don't know how to ask GitHub reviews to show me the commit where this file was introduced (I'm not on a dev machine so don't have access to the repo locally).  I'm guessing the commit message would say where these icons came from and why they were committed.

So that's essentially my question: are these icons assets that we should commit into our repo or should they be pulled in as some sort of dependency?  Sorry if this is a dumb question.  I was hoping to figure this out on my own by reading commit messages in GitHub but couldn't see a good way to do that.

Reply:
I suppose it would be a stylistic question.  If we made the SVG a separate file and simply imported it into this component it would only result in more to manage.  These SVG's can be custom or come from some free source like daisyUI or material icons... if you'd like to document the source on these, that is certainly something we can do in these files as a comment.

Reply:
Yes ideally we document in the file itself where the code came from, if we didn't write it ourselves (attribution)